modules/ztp-image-based-upgrade-rollback.adoc

```diff
@@ -27,21 +21,11 @@ spec:
   seedImageRef:
     version: 4.15.2
     image: <seed_container_image>
-  additionalImages:
-    name: ""
-    namespace: ""
-  autoRollbackOnFailure: {} <1>
-#    disabledForPostRebootConfig: "true" <2>
-#    disabledForUpgradeCompletion: "true" <3>
-#    disabledInitMonitor: "true" <4>
-#    initMonitorTimeoutSeconds: 1800 <5>
+  autoRollbackOnFailure: {}
```

Comment:
Be prepared to remove autoRollbackOnFailure - the plan is to move it to annotation. cc @donpenney

Reply:
I removed the options, then added the annotations but then removed the annotations too as per @pixelsoccupied 's comment [here](https://github.com/openshift/openshift-docs/pull/74025#discussion_r1570966574). Based on that thread, the `autoRollbackOnFailure.initMonitorTimeoutSeconds` remains so users can still configure the time limit.